models/User/AbstractUser.php

```diff
@@ -260,7 +261,11 @@ public function delete()
 
         // now delete the current user
         $this->getDao()->delete();
-        \Pimcore\Cache::clearAll();
+        
+        $cacheKey = 'user_' . $this->getId();
+        if (RuntimeCache::isRegistered($cacheKey)) {
```

Comment:
should we also clear parent cache(if exists) here? as this examples generates wrong result:
```
        $parent = User\Folder::getById($id);
        $parent->getChildren(); //count: 1
        
        $child = User::getById($cid);
        $child->delete();
        
        $parent = User\Folder::getById($id);
        $parent->getChildren(); //count: 1
```